src/django_components/templatetags/component_tags.py

```diff
@@ -62,83 +65,89 @@ def get_components_from_preload_str(preload_str: str) -> List["Component"]:
     else:
         rendered_dependencies = []
-        for component in get_components_from_registry(component_registry):
+        for component in _get_components_from_registry(component_registry):
             rendered_dependencies.append(component.render_dependencies())
 
         return mark_safe("\n".join(rendered_dependencies))
 
 
 @register.simple_tag(name="component_css_dependencies")
-def component_css_dependencies_tag(preload: str = "") -> SafeString:
+def component_css_dependencies(preload: str = "") -> SafeString:
     """Marks location where CSS link tags should be rendered."""
 
     if is_dependency_middleware_active():
         preloaded_dependencies = []
-        for component in get_components_from_preload_str(preload):
+        for component in _get_components_from_preload_str(preload):
             preloaded_dependencies.append(RENDERED_COMMENT_TEMPLATE.format(name=component.registered_name))
         return mark_safe("\n".join(preloaded_dependencies) + CSS_DEPENDENCY_PLACEHOLDER)
     else:
         rendered_dependencies = []
-        for component in get_components_from_registry(component_registry):
+        for component in _get_components_from_registry(component_registry):
             rendered_dependencies.append(component.render_css_dependencies())
 
         return mark_safe("\n".join(rendered_dependencies))
 
 
 @register.simple_tag(name="component_js_dependencies")
-def component_js_dependencies_tag(preload: str = "") -> SafeString:
+def component_js_dependencies(preload: str = "") -> SafeString:
     """Marks location where JS script tags should be rendered."""
 
     if is_dependency_middleware_active():
         preloaded_dependencies = []
-        for component in get_components_from_preload_str(preload):
+        for component in _get_components_from_preload_str(preload):
             preloaded_dependencies.append(RENDERED_COMMENT_TEMPLATE.format(name=component.registered_name))
         return mark_safe("\n".join(preloaded_dependencies) + JS_DEPENDENCY_PLACEHOLDER)
     else:
         rendered_dependencies = []
-        for component in get_components_from_registry(component_registry):
+        for component in _get_components_from_registry(component_registry):
             rendered_dependencies.append(component.render_js_dependencies())
 
         return mark_safe("\n".join(rendered_dependencies))
 
 
 @register.tag("slot")
-def do_slot(parser: Parser, token: Token) -> SlotNode:
-    # e.g. {% slot <name> ... %}
-    tag_name, *args = token.split_contents()
-    slot_name, is_default, is_required, slot_kwargs = _parse_slot_args(parser, args, tag_name)
-    # Use a unique ID to be able to tie the fill nodes with components and slots
-    # NOTE: MUST be called BEFORE `parser.parse()` to ensure predictable numbering
-    slot_id = gen_id()
-    trace_msg("PARSE", "SLOT", slot_name, slot_id)
+def slot(parser: Parser, token: Token) -> SlotNode:
```

Comment:
For the review, I suggest to open this file separately, to get a better feel for the changes. Primarily, I cleaned up the tag functions. Instead of doing custom parsing in each tag, there's now [`_parse_tag`](https://github.com/EmilStenstrom/django-components/pull/572/files#diff-7345171ed38e1dd2e4b8199b65a2fd082323bdb331ff0ef0dc0dd3bd43f111aeR335), which handles it all.

So all tags (`component`, `slot`, `fill`, `provide`, ...) now offer the same behavior - using `/` for inline tags, handling the `kwarg:key=val` syntax, etc.

This doesn't apply to the template tags that deal with the JS/CSS dependencies, as those might yet change when we work on that.

---

Also, I updated the tag names to match the tag names.